### PR TITLE
Update 2017-11-11-environments.md

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

**Comment by orbeckst** [comment]
Why the date 2017-11-11? 

Otherwise looks fine to me.

**Comment by orbeckst** [comment]
... or is this an alternative for PR #47 ?

**Comment by richardjgowers** [comment]
This is a PR to the PR. I didn't want to directly edit Mac's post this much without his approval

**Comment by kain88-de** [comment]
> Why the date 2017-11-11?

Makes the post appear at the top on my local builds of the website. And it's  far enough in the future that I'm likely seeing that I need to change the date. If you know another good way to prepare blog posts and render them as a preview I'm all ears.

File: _posts/2017-11-11-environments.md [hunk 1]
```diff
@@ -1,62 +1,75 @@
 ---
 layout: post
-title: Working with different package versions
+title: Managing software versioning using Conda environments
```
Comment (kain88-de) [inline _posts/2017-11-11-environments.md]: I wouldn't stick to just conda here. For some people that would mean switching their complete python stack (and most scientists I know are averse to switching working systems). The idea behind the post is to show people how they can use newer version of MDAnalysis while still keeping their old scripts running.

**Comment by richardjgowers** [comment]
I thought you can add conda on top of an existing stack though? Ie the
default environment will be what they previously had.

**Comment by kain88-de** [comment]
No you can add conda on top. You would switch completely. 

If you install conda and switch to it you will change your global python installation and keep everything that is installed in `~/.local/lib/pythonX.X`. The local installation is  a problem though for compiled packages like MDAnalysis, here the version of numpy we compile MDAnalysis against has to match the global version.  So for a switch to conda I would recommend to make a list of packages in `~/.local/lib/pythonX.X` and install them from conda-forge or reinstall from pip.

**Comment by orbeckst** [comment]
https://github.com/MDAnalysis/MDAnalysis.github.io/pull/48#issuecomment-290018491



>>    Why the date 2017-11-11?
>
> Makes the post appear at the top on my local builds of the website. And it's far enough in the future that I'm likely seeing that I need to change the date. If you know another good way to prepare blog posts and render them as a preview I'm all ears.


Perhaps jekyll drafts works for you: https://jekyllrb.com/docs/drafts/

**Comment by orbeckst** [comment]
> No you can add conda on top. You would switch completely.

What works for me is to use `virtualenv` for most work but also have a conda environment (namely for Python 3) that I activate on demand with a bash function
```bash
function anaconda3 ()
{
    export PATH=${HOME}/anaconda3/bin:$PATH;
    echo "Using anaconda3..."
}
```

**Pull request merged**